inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/region_yolo_gpu_ref.cl

```diff
@@ -12,94 +12,79 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "include/common.cl"
-#include "include/data_types.cl"
-
-#define IW INPUT0_SIZES[0]
-#define IH INPUT0_SIZES[1]
-#define IC INPUT0_SIZES[2]
-#define IB INPUT0_SIZES[3]
+#include "include/include_all.cl"
 
 inline UNIT_TYPE FUNC(logistic_activate)(UNIT_TYPE x) {
     return 1. / (1. + exp(-x));
 }
 
-inline int FUNC(entry_index)(int width, int height, int coords, int classes,
-                       int outputs, int batch, int location,
-                       int entry) {
-    int n = location / (width * height);
-    int loc = location % (width * height);
-    return batch * outputs + n * width * height * (coords + classes + 1) +
-        entry * width * height + loc;
+inline int FUNC(output_index)(int batch, int region_num, int x, int y, int xy, int feature_offset) {
+#if DO_SOFTMAX
+    return OUTPUT_GET_INDEX(batch, feature_offset * INPUT0_SIZE_X * INPUT0_SIZE_Y + xy, 1, 1);
+#else
+    return OUTPUT_GET_INDEX(batch, feature_offset, y, x);
+#endif
 }
 
-#if DO_SOFTMAX
-inline void FUNC(softmax_generic)(const __global UNIT_TYPE* src_data, __global UNIT_TYPE* dst_data,
-                            int B, int C, int W, int H, int i)
+KERNEL (region_yolo_ref)(const __global UNIT_TYPE* input, __global UNIT_TYPE* output)
```

Comment:
`UNIT_TYPE` is deprecated. Please use `INPUT0_TYPE` and `OUTPUT_TYPE`

Reply:
Yes, I'll update it.